sites/example-project/vite.config.js

```diff
@@ -8,7 +8,12 @@ const config = {
 		exclude: ['svelte-icons', 'svelte-tiny-linked-charts']
 	},
 	ssr: {
-		external: ['@evidence-dev/db-orchestrator', '@evidence-dev/telemetry', 'blueimp-md5']
+		external: [
+			'@evidence-dev/db-orchestrator',
+			'@evidence-dev/telemetry',
+			'blueimp-md5',
+			'@evidence-dev/plugin-connector'
```

Comment:
What changed in plugin connector that requires it to go in ssr external now?

Reply:
I think this is related to the dynamic imports above, need to go back and look